src/main/java/core/basesyntax/service/impl/WriterImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Writer;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriterImpl implements Writer {
+    @Override
+    public void writeToFile(String report, String filePath) {
+        File file = new File(filePath);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file))) {
+            bufferedWriter.write(String.valueOf(report));
```

Comment:
any reason for that?
```suggestion
            bufferedWriter.write(report);
```